app/models/contact.rb

```diff
@@ -28,6 +28,14 @@ def needs_primary_email?
     ['Client', 'Volunteer'].include? contactable_type
   end
 
+  def validate_first_name?
+    contactable_type != 'Department' && contactable_type != 'Profile'
```

Comment:
I did it exactly like that at first, but it just behaved strange on me - didn't happen to work.
But no problem, I can try again.